experimental/cloudstrapper/playbooks/roles/vars/cluster.yaml

```diff
@@ -0,0 +1,35 @@
+
+#------------------
+#Cluster-specific Variables
+#Use them to customize each deployment
+#------------------
+#Orc8r deployment variables
+orc8rClusterName: Seoul
+orc8rDomainName: arun.fbmagma.ninja
+orc8rLabel: 1.3.0
+orc8rVersion: v1.3
+gitHelmRepo: helm13
```

Comment:
minor nit: Can you probably add a comment for a repo 
https://github.com/helm13/helm13.git, use the repo name "helm13" here.

Reply:
Added a reference to clarify how to set gitHelmRepo name.